renderdoc/driver/vulkan/vk_stringise.cpp

```diff
@@ -247,6 +247,10 @@ rdcstr DoStringise(const VulkanChunk &el)
     STRINGISE_ENUM_CLASS(vkCreateRayTracingPipelinesKHR)
     STRINGISE_ENUM_CLASS(vkCmdSetRenderingAttachmentLocationsKHR)
     STRINGISE_ENUM_CLASS(vkCmdSetRenderingInputAttachmentIndicesKHR)
+    STRINGISE_ENUM_CLASS(vkCopyImageToImage)
+    STRINGISE_ENUM_CLASS(vkCopyImageToMemory)
+    STRINGISE_ENUM_CLASS(vkCopyMemoryToImage)
+    STRINGISE_ENUM_CLASS(vkTransitionImageLayout)
```

Comment:
It doesn't particularly have to but for slight clarity benefits this should match the enum order.

Reply:
Done